variables.tf

```
@@ -130,3 +130,15 @@ variable "login_service_accounts" {
   type        = list(string)
   default     = []
 }
+
+variable "fw_allowed_cidrs" {
+  description = "List of source IP address of particular range to which the firewall rule will apply. These ranges must be expressed in CIDR format."
+  type        = list(string)
+  default     = ["35.235.240.0/20"]
```

If `"35.235.240.0/20"` is part of the default, then what happens if anyone overrides this variable? They will have to remember to add this value into their overriding value.

Instead, this should be abstracted inside the module so that it's always present whether this variable is passed with empty list or filled list. Check my comments above.

```suggestion
  default     = []
```